modules/core/keeper/msg_server.go

```diff
@@ -286,7 +286,7 @@ func (k Keeper) ChannelOpenInit(goCtx context.Context, msg *channeltypes.MsgChan
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, connectiontypes.AttributeValueCategory),
+			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
```

Comment:
`channeltypes` is already alias imported! 

```suggestion
			sdk.NewAttribute(sdk.AttributeKeyModule, channeltypes.AttributeValueCategory),
```

Reply:
Hmm yea, the other channel callbacks just referenced `types.AttributeKeyModule` so I kept it consistent (this references the correct key) but it'd probably be cleaner to update them all to be `channeltypes`.

Reply:
I see that we import the channel types twice?

```
"github.com/cosmos/ibc-go/v2/modules/core/04-channel/types"
channeltypes "github.com/cosmos/ibc-go/v2/modules/core/04-channel/types"
```

Reply:
Yeah, saw that as well. I'll refactor the other callbacks to use channeltypes and remove the other import (this was not added by me)